addon/components/menu/items.hbs

```diff
@@ -1,5 +1,5 @@
 {{! template-lint-disable no-down-event-binding }}
-{{#if @isOpen}}
+{{#if (or @isOpen @static)}}
```

Comment:
do we need `@static`? I really don't like over-expanding public API, and it seems just setting `@isOpen={{true}}` would be sufficient?

Reply:
It seems the other implementations support a `static` option as well.

Reply:
Yes I included the addition of the `static` option as it is supported in the [React & Vue implementations](https://headlessui.dev/react/menu#menu-items) and is already available in the [ListBoxOptions](https://github.com/GavinJoyce/ember-headlessui/blob/master/addon/components/listbox/-options.hbs) component

In terms of a use case we will be using multiple Menus in a left nav and we need don't require the dropdown to close when focus is lost, only if the user has clicked to close the Menu, or another Menu has been opened. Utilising the `static` option and to override the behavior in our app seemed like the best way to achieve this.

Reply:
Regarding use case:
Somewhere in react doc, I saw information that might be useful for handling with own transition and animation.

Reply:
You're right it's [here](expandedNav) thank you!

Reply:
What's the advantage over using isOpen though? I'm confused. Are the semantics really that much better even though the arg does the exact same thing?
Is there a difference in the other implementation?
🤔

Reply:
hmm I think that `<Menu.Items/>` does not expect to receive `isOpen` prop. It is in template because is passed down from `<Menu/>` component, but intention was to not override this prop.
[Here ](https://headlessui.dev/react/menu#showing-hiding-the-menu)

> By default, your Menu.Items instance will be shown/hidden automatically based on the internal open state tracked within the Menu component itself.

Probably it is possible to do it without `static` arg, but then the question is how far we want to be from original API in react and vue

Reply:
ah! so `@isOpen` is actually private api. 

I think perhaps I need to document the public/private ness of all the args 😅